tools/parse_collection.cpp

```diff
@@ -1,182 +1,16 @@
 #include <string>
 
 #include <CLI/CLI.hpp>
-#include <KrovetzStemmer/KrovetzStemmer.hpp>
-#include <Porter2.hpp>
-#include <boost/algorithm/string.hpp>
 #include <fmt/format.h>
 #include <spdlog/spdlog.h>
 #include <tbb/task_scheduler_init.h>
-#include <trecpp/trecpp.hpp>
-#include <wapopp/wapopp.hpp>
-#include <warcpp/warcpp.hpp>
 
 #include "forward_index_builder.hpp"
+#include "query/term_processor.hpp"
+#include "record_parser.hpp"
```

Comment:
Should this be `parser.hpp` or am I missing something here?

Reply:
Thanks!